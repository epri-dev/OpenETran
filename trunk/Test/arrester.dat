1 3 30.0 1 1 0.001e-6 5.0e-6

cable 1 400.0 3.0e8 0.0e3

surge 20.0e3 1.0e-6 50.0e-6 0.0e-6
pairs 1 0
poles 2

arrester -37.2e3 37.2e3 0.28 1.0e-6 0.0
pairs 1 0
poles 2

meter
pairs 1 0
poles 2


