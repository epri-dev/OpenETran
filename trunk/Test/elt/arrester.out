  N   span     dT   Tmax
  1  30.00 1e-009 5e-006
Z-phase
  4.00000e+002
Z-modal
  4.00000e+002
Modal Transformation (Ti)
  1.00000e+000
Power-frequency source at pole 1.
   #        vp        vm         i
   1       0.0       0.0       0.0
Power-frequency source at pole 3.
   #        vp        vm         i
   1       0.0       0.0       0.0
Meter at pole 2, from 1 to 0, max V   = 4.274016e+004
Meter at pole 2, arrester max I       = 1.978628e+004
Arrester at pole 2, from 1 to 0 discharged 1.978628e+004 Amperes.
	Time operated: 8.700000e-008	Time of peak: 1.408000e-006
	Charge: 8.308498e-002	Energy: 3.522405e+003
nr_iter = 0, nr_max = 0
Execution time [sec]: 0.031
